PopTop.podspec

```diff
@@ -12,8 +12,8 @@ Pod::Spec.new do |s|
   s.author       = { 'AJ Self': 'aj.self3@gmail.com' }
   s.platform     = :ios, '8.0'
 
-  s.source       = { git: 'https://02e34411c57bedacd7b5e66d60efaa3b9ffbc346@github.com/bellycard/PopTop.git', tag: s.version.to_s }
```

Comment:
AJ found that GitHub automatically revoked the token when I flipped the open-source switch.